priv/www/js/formatters.js

```diff
@@ -567,6 +567,16 @@ function fmt_shortened_uri(uri) {
     }
 }
 
+function fmt_uri_with_credentials(uri) {
+    if (typeof uri == 'string') {
+        // mask password
+        var mask = /^([a-zA-Z0-9]+):\/\/(.*):(.*)@/;
```

Comment:
The only URI schemes we'll run into at this point are `amqp` and `amqps`, but it appears that a couple other characters are valid in them: `-`, `+` and `.` ([RFC](http://www.ietf.org/rfc/rfc2396.txt)).

Something like this -

`^([a-zA-Z0-9-+.]+):\/\/([^:]*):([^@]*)@`